source/_integrations/zwave_js.markdown

```diff
@@ -460,6 +460,44 @@ In addition to the [standard automation trigger data](/docs/automation/templatin
 | `trigger.current_value`      | The current value for this Z-Wave value (translated to a state name when possible).        |
 | `trigger.current_value_raw`  | The raw current value for this Z-Wave value (the key of the state when a state is named).  |
 
+### `zwave_js.event`
+
+This trigger platform can be used to trigger automations on any Z-Wave JS event, including events that may not be handled by Home Assistant automatically.
```

Comment:
Should we link to the Z-Wave JS api docs here?

Reply:
good idea